pom.xml

```diff
@@ -151,6 +151,10 @@
             <groupId>org.springdoc</groupId>
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
+        </dependency>
```

Comment:
Same as for loadflow, it would be better to import spring-boot-actuator as compiled

Reply:
Done